src/scp/SCPDriver.cpp

```diff
@@ -67,6 +67,12 @@ SCPDriver::toStrKey(NodeID const& pk, bool fullKey) const
     return fullKey ? KeyUtils::toStrKey(pk) : toShortString(pk);
 }
 
+NodeID
+SCPDriver::fromStrKey(std::string const& pkStr) const
+{
+    return KeyUtils::fromStrKey<NodeID>(pkStr);
```

Comment:
should we also update https://github.com/stellar/stellar-core/blob/89b1dcd20e61d1c0970ffa589da6913220554e76/src/main/CommandHandler.cpp#L1145C5-L1145C5 to use this new helper?

Reply:
Good idea. `CommandHandler` doesn't have access to an `SCPDriver`, so I added a function to `Herder` that exposes the `fromStrKey` function in its `HerderSCPDriver` and called that from `CommandHandler::surveyTopology` (see 2dec3aefcb8f09940b43add26887e2b69586f400)

Reply:
Thanks! I think the new helper should live in KeyUtils then, since it's not related to SCPDriver (it should also be static/free function)

Reply:
Done in 2b2ecb16c119848fd26b86ec60dd20601fe61e04